.golangci.yaml

```diff
@@ -48,7 +48,6 @@ issues:
 output:
   sort-results: true
 run:
-  concurrency: 8
```

Comment:
high concurrency can faster process time, the default is 4, if you want, we can test this on this PR, but need a `.go` file change to trigger the job

Reply:
docu says:   `# The default concurrency value is the number of available CPU.`